### PR TITLE
MAILBOX-342 Mutualize Cassandra ManageSieve MPT tests

### PR DESCRIPTION
**Pull request by chibenwa**
Reuse valuable changes from https://github.com/linagora/james-project/pull/1571

### CODE REVIEW

File: mpt/impl/managesieve/cassandra/src/test/java/org/apache/james/mpt/managesieve/cassandra/CassandraManageSieveMPTTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import org.apache.james.mpt.testsuite.RenameScriptContract;
+import org.apache.james.mpt.testsuite.SetActiveContract;
+import org.apache.james.mpt.testsuite.StartTlsContract;
+import org.apache.james.mpt.testsuite.UnauthenticatedContract;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(DockerCassandraExtension.class)
+class CassandraManageSieveMPTTest implements AuthenticateContract, CapabilityContract, CheckScriptContract,
```
Comment (mbaechler) [inline mpt/impl/managesieve/cassandra/src/test/java/org/apache/james/mpt/managesieve/cassandra/CassandraManageSieveMPTTest.java]: what about defining a ManageSieveMPTContract interface that extends all the sub-contrats ?

Comment (chibenwa) [reply mpt/impl/managesieve/cassandra/src/test/java/org/apache/james/mpt/managesieve/cassandra/CassandraManageSieveMPTTest.java]: Seems like a good idea.

**Comment by chibenwa** [comment]
```
[INFO] Apache James MPT ManageSieve Cassandra ............. SUCCESS [ 17.762 s]
```

Before was : 

```
[INFO] Apache James MPT ManageSieve Cassandra ............. SUCCESS [03:16 min]
```

Failed while dockering JPA Guice. Due to weak dockerfile isolation (this PR isNOT rebased on master hence incorrect JPA java-agant version is used...) Can be considered :green_apple:

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**